src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -170,12 +198,31 @@ public class MethodologyService : IMethodologyService
 
         private async Task<Either<ActionResult, Unit>> ValidateMethodologySlugUniqueForUpdate(Guid id, string slug)
         {
-            if (await _context.Methodologies.AnyAsync(methodology => methodology.Slug == slug && methodology.Id != id))
+            var methodologyParentId = await _context
+                .Methodologies
+                .Where(m => m.Id == id)
+                .Select(m => m.MethodologyParentId)
+                .SingleAsync();
+            
+            if (await _context
+                .MethodologyParents
+                .AnyAsync(p => p.Slug == slug && p.Id != methodologyParentId))
             {
                 return ValidationActionResult(SlugNotUnique);
             }
 
             return Unit.Instance;
         }
+        
+        private async Task<Either<ActionResult, Methodology>> HydrateMethodologyForManageMethodologySummaryViewModel(
+            Methodology methodology)
+        {
+            await _context
```

Comment:
I think it might be a good idea to comment this to mention it's been done to assist the `Methodology.Title` method which is deferring to the MethodologyParent if no alternative title is set. That's if my detective work about why this is needed is correct! 🙂 

Also same comment for this method in the `MethodologyContentService`

Reply:
OK!

Reply:
Done.